src/parser/core/modules/Timeline/Timeline.module.css

```diff
@@ -17,6 +17,27 @@
 	height: 100%;
 }
 
+:global .vis-range .vis-item-overflow {
+	padding: 0;
+}
+
+:global .vis-item.vis-background .vis-item-content {
+	width: 100% !important;
+}
+
+:global .vis-range .vis-item-content {
+	padding: 0;
+}
+
+:global .vis-range .progress-fl img {
```

Comment:
medium; doesn't look like `.progress-fl` is being used anywhere now.

Reply:
done